ergo/foretold.py

```diff
@@ -0,0 +1,70 @@
+from matplotlib import pyplot as plt
+import torch
+import numpy as np
+import requests
+from ergo.ppl import uniform
+
+
+class ForetoldDistribution:
+    """"Aggregated distribution from a foretold question"""
+
+    def __init__(self, id):
+        """
+            id: measurableId, the second id in the URL for a foretold question
```

Comment:
maybe add a quick explanation of why we'd expect this to be true given how the Foretold distribution is set up?

and/or add a link to the foretold question?

Reply:
Done